app/assets/javascripts/discourse/templates/components/categories-boxes-with-topics.hbs

```diff
@@ -11,6 +11,7 @@
             {{#if c.read_restricted}}
               {{d-icon 'lock'}}
             {{/if}}
+            {{plugin-outlet name="category-title-before" noTags=true args=(hash category=c)}}
```

Comment:
Let's move this into a component so that we don't have to duplicate the same line 3 times. :smile:

Reply:
Done.